### ISSUE TITLE
E3, N3 - Teaser List RSS Feeds do not update alternative text when filtered by category

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#549]
Teaser List RSS Feeds do not update alternative text when filtered by category. This is a WCAG 2.0 failure as we have duplicate link text going to different URLs.

![image](https://cloud.githubusercontent.com/assets/1927902/19170397/3d178120-8be6-11e6-8805-81a6fb39a98f.png)

**Comment by tqureshi-uog** [comment]
https://www.drupal.org/node/2166307 may have solution

**Comment by tqureshi-uog** [comment]
Actual solution: under "Advanced" options for the Feed view, click "Contextual filters" and then, under "When the filter value IS available or a default is provided" check "Override title" and choose the category token %1 as the override value. This will ensure the Feed title will be the same as the category instead of the default of "News", "Upcoming Events", or "Features". The RSS icon alt text pulls the feed title, so as long as the title is distinct, the alt text will be as well.

**Note:** In addition to N3 and E3, I made the same change to FT3 since it would run into the same issue if used twice on the same page.
